### ISSUE TITLE
Make the DoorBot RFID boards more secure

### ISSUE THREAD

**Issue by johnmckerrell** [DoESLiverpool/somebody-should#1267]
Doorbot5 stopped working this morning as the RFID circuit board had fallen into its box. Rather than sticking the board to the front I wondered about 3D printing something to go behind and push it to the front. Possibly some 3D printed spring-type mechanism which could be compressed but would continuously push forwards (lightly)?

Would be worth prototyping on one of the internal fob readers and if it seems to work and doesn't push the box open then use on all of the fobs readers.

Not having much experience with modelling stuff for 3D printing I'm not expecting to do this one myself.

The board that we're pushing to the front is 38x38mm and I'll post a photo shortly.

**Comment by johnmckerrell** [comment]
![2019-09-30 16 44 55](https://user-images.githubusercontent.com/27243/65896183-ad1cd380-e3a4-11e9-9ff0-670c359fb238.jpg)

As you'll see there's only one hole, I think it would be enough to have a small square frame to wrap the the board and then whatever springed legs we could manage coming off the back to push it forwards.

**Comment by Sean-anotherone** [comment]
chunk of foam rubber?

**Comment by johnmckerrell** [comment]
That's a very nice idea, I wonder where @jackie1050 put the spare stuff we had lying around.

**Comment by jackie1050** [comment]
In the bin - unfortunately that's how last chance works!

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by johnmckerrell** [comment]
Fair enough, I couldn’t remember if you’d put it away somewhere because you wanted it!

**Comment by jackie1050** [comment]
There's some foam in the laser cutter room - it's less springy but might still do the job

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
